.github/workflows/release_note.py

```diff
@@ -210,36 +212,95 @@ def generate_package_update_section(version):
                     output += _parse_match_section(s['matchBinaries'])
     return output
 
+def release_notes_changes_section():
+    return textwrap.dedent("""
+    ## Changes
+    The following packages have been upgraded, to address the mentioned CVEs:
+    **todo release facilitator: fill this in**
+    """)
 
-def create_github_release_notes(gardenlinux_version, commitish):
-    commitish_short=commitish[:8]
-    output = ""
-  
-    manifests = download_all_singles(gardenlinux_version, commitish_short)
-
-    output += generate_release_note_image_ids(manifests)
-
-    output += "\n"
+def release_notes_software_components_section(package_list):
+    output = "\n"
     output += "## Software Component Versions\n"
     output += "```"
     output += "\n"
+    packages_regex = re.compile(r'^linux-image-amd64$|^systemd$|^containerd$|^runc$|^curl$|^openssl$|^openssh-server$|^libc-bin$')
```

Comment:
What's with the packages targeting `arm64`? Is there are reason we exclude them?